CHARTER.md

```diff
@@ -0,0 +1,127 @@
+# MapLibre Charter
+
+## Definitions
+
+*MapLibre Organization:* An umbrella organization to develop, maintain and promote MapLibre software projects
+
+*MapLibre Projects:* multiple repositories, each being some part of the map visualization stack from the raw data to the map visualization and interaction.
+
+*MapLibre:* As a single word, “MapLibre” is a brand name used by the front-end map rendering MapLibre projects (i.e. code libraries).
+
+*MapLibre Community:* People interested in the MapLibre Projects. Anyone can be part of the MapLibre Community and there are no special requirements except to follow the Community Code of Conduct. For example, the Community members … :
+
+ * evangelize about the MapLibre Organization and its Projects
+ * educate new users on the project’s strengths and weaknesses
+ * provide moral support (a ‘thank you’ goes a long way)
+ * support the MapLibre Organization financially
+ * improve code and fix bugs
+ * help with documentation
+ * give advice to others
+
+*Technical Steering Committee:* An informal group of people who meet on a monthly basis and discuss technical topics related to software development. Everyone is welcome to join the Technical Steering Committee.
+
+*Voting Member:* A person who can elect the Governing Board and vote on changes to the MapLibre Charter.
+
+*Governing Board:* The elected representatives of the MapLibre Community who help shape the Project's overall direction and who are responsible for the Project's finances.
+
+*Third-party Organization:* A company or organization that uses MapLibre software for commercial or non-commercial purposes.
+
+## MapLibre Organization Goals
```

Comment:
nit: Formatting changed here; previous headline was followed by a blank line (which is probably better?). It also changes again further down below.

Reply:
Markdownlint likes newlines, fwiw. Probably worth running it through that before finalizing to avoid people wanting to make changes after this has been voted on.

Reply:
Good idea... https://github.com/maplibre/maplibre/pull/99